package.json

```diff
@@ -15,6 +16,6 @@
     "mocha-junit-reporter": "1.6.1"
   },
   "scripts": {
-    "test": "grunt verify --force"
+    "test": "./node_modules/mocha/bin/mocha test/generic.spec.js"
```

Comment:
@kbuchacz can you document somewhere (wiki) what we allow and point @jbeynar to that document?